src/backend/TrafficCourts/Citizen.Service/Validators/Rules/CountActRegMustBeMVA.cs

```diff
@@ -3,12 +3,11 @@
 namespace TrafficCourts.Citizen.Service.Validators.Rules;
 
 /// <summary>
-/// For all 3 counts, the ACT/REGs section must be "MVA" or blank.
+/// For all 3 counts, the ACT/REGs section must be "MVA", "MVR", or blank.

```

Comment:
As far as I understood from the requirements and acceptance criteria, ACT/REGs section must not be blank anymore. Must have "MVA" or "MVR" written in that box to count as MVA ticket. So should we update the if check below based on that to not allow blank box?

Reply:
I had the same question, but then I realized that not all 3 counts must be filled in - some tickets may only have 1 or 2 counts. In such cases this field will be blank on the blank count row.

Reply:
Ah I see, that makes sense.